### ISSUE TITLE
NSM support?

### ISSUE THREAD

**Issue by mxmilkiib** [DISTRHO/Cardinal#49]
Dependent on https://github.com/DISTRHO/DPF/issues/137 I think.

**Comment by falkTX** [comment]
Yes, and until that tool becomes more useful this request goes nowhere for now.
